slides/week2/week2.org

```diff
@@ -0,0 +1,82 @@
+#+TITLE: Week 2
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* What are we doing today?
+- Introduction to C++
+ - What it is
+ - Functions
+ - Includes and headers
+ - Working with strings
+- How to find and read documentation
+- Sending and recieving changes from remote Git repositories
+
+* What is C++?
+- A general-purpose programming language
+- Designed with performance, efficiency, and flexibility in mind
+- Great for robots!
+
+* Functions
+#+BEGIN_NOTES
+After introducing the concept, write a simple example. Don't go into prototypes yet, we'll cover them in a bit.
+
+TODO I think we should design a specific example that is easy to code on the fly but isn't completely trivial (e.g. writing an adder function seems dumb)

```

Comment:
maybe a function that takes in a char and a number, and prints out said number of chars. 

`*`, `5` -> `*****`.

Not sure how easy you're looking for though.

You could expand on the above function to build a triangle printer (for nested functions):

```
*
**
***
****
```